src/index.html

```diff
@@ -4,15 +4,63 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <title>Stars</title>
+    <title>Star Blocks</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div>
+      <div class="stars--0 stars__container">
+        <div class="stars"></div>
+        <div class="stars"></div>
+        <div class="stars"></div>
+        <div class="stars"></div>
+        <div class="stars"></div>
+      </div>
```

Comment:
please, follow bem conventions

use `stars` as a name for the container
use `stars__star` as a class for the each star
use `stars--1` and so on, modifier to set amount of highlighted stars